httpx/middleware/request_id.go

```diff
@@ -42,6 +42,13 @@ func (h *RequestID) ServeHTTPContext(ctx context.Context, w http.ResponseWriter,
 	return h.handler.ServeHTTPContext(ctx, w, r)
 }
 
+func AddRequestIDToCtx(h httpx.Handler) httpx.Handler {
```

Comment:
Ah, I see. This is so you can provide the `func(httpx.Handler) httpx.Handler` signature. Brings up my point in https://github.com/remind101/pkg/pull/28/files#r51063405.